src/sentry/static/sentry/app/views/performance/trends/chart.tsx

```diff
@@ -55,6 +61,38 @@ function transformEventStats(data: EventsStatsData, seriesName?: string): Series
   ];
 }
 
+function transformEventStatsSmoothed(data: Series[], seriesName?: string): Series[] {
+  const currentData = data[0].data;
+  const resultData = [] as SeriesDataUnit[];
+  const firstValue = movingAverage(currentData, AVERAGE_WINDOW / 2, AVERAGE_WINDOW);
+  const lastValue = movingAverage(
+    currentData,
+    currentData.length - AVERAGE_WINDOW / 2,
+    AVERAGE_WINDOW
+  );
+
+  for (let i = 0; i < currentData.length; i++) {
```

Comment:
Should you start at the tail of the results so that the last window is complete? Right now you will have partial windows at the head and tail of the results. Alternatively you could start at the 10th element so that you're always working with full windows.

Reply:
So this ties back to Alberto's comments [here](https://github.com/getsentry/sentry/pull/21271#discussion_r503241557)
I wanted to avoid gaps at the beginning/end (now just beginning since moving average is correct) so i start at 0 to set the first average across the first 20 values.